### ISSUE TITLE
[Bug]: Parts of the Project Frame are transparent when you set a background image

### ISSUE THREAD

**Issue by EricMcWinNer** [mallowigi/material-theme-issues#332]
### Describe the bug

When the material theme project frame is used with a background image, and the opacity is reduced, it also makes parts of the project frame transparent and makes reduces the quality of the UI.

As seen in the screenshots I logged, some parts of the project frame are actually opaque, like the section with the breadcrumb at the top-left and some of the buttons at the right, but the project title and the rest of the body are transparent and makes it aesthetically unpleasing. 

The project frame would look a lot better for users who code with background images if he entirety of the project frame was opaque and doesn't respond to the background image settings on the rest of the IDE.

![Screenshot 2023-04-09 at 4 40 39 AM](https://user-images.githubusercontent.com/19782345/230752942-67434be9-918d-437d-8cb1-76b3bd919b71.png)

![Screenshot 2023-04-09 at 4 42 00 AM](https://user-images.githubusercontent.com/19782345/230752987-96c398d9-3003-4c22-af60-f4a775d390c1.png)

### To Reproduce

1. Install Material Theme UI
2. Activate the project frame
3. Set a background image and make the image visible by reducing the opacity.

### Expected behavior

The entire project frame should be opaque and shouldn't show the background image set underneath

### Material Theme Version

v8.6.2

### IDE

PHPStorm

### System Info

```shell
PhpStorm 2023.1
Build #PS-231.8109.199, built on March 29, 2023
Licensed to Eric Aprioku
Subscription is active until October 4, 2023.
Runtime version: 17.0.6+10-b829.5 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 13.2.1
GC: G1 Young Generation, G1 Old Generation
Memory: 4504M
Cores: 8
Metal Rendering is ON
Registry:
    editor.focus.mode.color.light=828bb8
    ide.editor.tab.selection.animation=true
    debugger.new.tool.window.layout=true
    run.processes.with.pty=TRUE
    ide.new.editor.tabs.vertical.borders=true
    ide.experimental.ui=true
    ide.balloon.shadow.size=0
    editor.focus.mode.color.dark=828bb8

Non-Bundled Plugins:
    io.unthrottled.amii (1.1.7)
    com.karateca.jstoolbox (1.10)
    com.intellij.ideolog (203.0.30.0)
    Key Promoter X (2023.1.0)
    com.tabnine.TabNine (1.0.4)
    indent-rainbow.indent-rainbow (2.1.0)
    com.wakatime.intellij.plugin (14.1.4)
    com.kalessil.phpStorm.phpInspectionsEA (4.0.7.1)
    ru.adelf.idea.dotenv (2023.1)
    net.king2500.plugins.PhpAdvancedAutoComplete (1.1.0)
    de.espend.idea.php.toolbox (6.2.0)
    de.espend.idea.php.annotation (9.2.0)
    izhangzhihao.rainbow.brackets (2023.2.6)
    dev.ekvedaras.laravelquery (4.0.2)
    fr.adrienbrault.idea.symfony2plugin (2022.1.246)
    com.chrisrm.idea.MaterialThemeUI (8.6.2)
    com.niclas-van-eyk.laravel-make-integration (3.5.0)
    com.dmarcotte.handlebars (231.8109.91)
    mdx.js (231.8109.91)
    com.laravel_idea.plugin (7.0.1.231)
    net.seesharpsoft.intellij.plugins.csv (3.1.0-231)
    com.mallowigi (90.0.0)
    zielu.gittoolbox (500.0.6+213)
    mobi.hsz.idea.gitignore (4.5.0)
    de.espend.idea.php.phpunit (7.1)
```

**Comment by mallowigi** [comment]
That's the first time I've heard of such a setup but from the looks of it, it doesn't look like this is feasible due to the nature of the background image opacity feature.

How does it look with the plugin disabled ?

**Comment by EricMcWinNer** [comment]
With the plugin disabled, the menubar is actually opaque as shown in the screenshot here.

It'd be cool if the project frame could also have the same feature with the per-project color for the frame.

I perfectly understand if it's not possible due to the way the Jetbrains background image opacity works, but it'd be a nice-to-have.


<img width="1195" alt="Screenshot 2023-04-09 at 9 54 36 AM" src="https://user-images.githubusercontent.com/19782345/230763484-72209765-5565-4b5b-b038-d2610b87944a.png">

**Comment by mallowigi** [comment]
I was rather talking about the project frame (I think I need to rethink the naming) rather than the title bar. 

I'll see if it's possible when I have the time.

PS: nice background

**Comment by EricMcWinNer** [comment]
Oh... I got confused since you said to test with the plugin disabled. The frame doesn't exist without the plugin. 

Anyways, I love this, Material Theme is like 55% of the reason I use PHPStorm. Thank you for all the hard work. 

I also love the project frame, I'm visual person and I work on multiple windows a lot, I used to find myself mixing up the projects some times, but this project frames with different colors completely solves that problem.

Thanks again for your hardwork. The community appreciates you.
